scripts/anvil_tools/README.md

```diff
@@ -24,13 +24,33 @@
     Locally
         `python3 /scripts/anvil_tools/add_user_to_workspace.py -t TSV_FILE [-p WORKSPACE_PROJECT]`
     Docker
-        `docker run --rm -it -v "$HOME"/.config:/.config -v "$HOME"/local_data_directory/:/data broadinstitute/horsefish:anvil_tools bash -c "cd data; python3 /scripts/anvil_tools/add_user_to_workspace.py -t /data/INPUT.tsv [-p WORKSPACE_PROJECT]"`
+        `docker run --rm -it -v "$HOME"/.config:/.config -v "$HOME"/local_data_directory/:/data broadinstitute/horsefish bash -c "cd data; python3 /scripts/anvil_tools/add_user_to_workspace.py -t /data/INPUT.tsv [-p WORKSPACE_PROJECT]"`
 
         Note: `local_data_directory` should be the path to the folder where your desired input .tsv file is located.
 ##### Flags
     1. `--tsv`, `-t`: input .tsv file (required)
     2. `--project`, `-p`: workspace project/namespace for listed workspaces in tsv (default = anvil_datastorage)
 
+#### **file_exists_checker.sh**
+##### Description
+    Check if list of files exist at denoted gs://XXXX path in GCS location.
+
+    Input is:
+        1. excel file with column of gs://XXXX file paths to validate (must end in .xlsx)
+        2. number of column with gs://XXXX file paths (must be an integer value)
+    Output is a .xlsx file with name:
+        1. `{input_excel_filenamme}_with_exists.xlsx`
+##### Usage
+    Locally
+        `./file_exists_checker EXCEL_FILE COLUMN_NUMBER_TO_CHECK`
+    Docker
+        `docker run --rm -it -v "$HOME"/.config:/.config -v "$HOME"/local_data_directory/:/data broadinstitute/horsefish bash -c "export CLOUDSDK_PYTHON=python2.7; cd data; ./../scripts/anvil_tools/file_exists_checker.sh EXCEL_FILE COLUMN_NUMBER_TO_CHECK; unset CLOUDSDK_PYTHON"`
```

Comment:
wait so why do we need to use python2 here?

Reply:
when running the script, looks like cloudsdk is using python3.9 which is giving an error about one of the imported packages it uses. the internet says to use python3.7 specifically so i just set the env variable for this script.

Reply:
`AttributeError: module 'base64' has no attribute 'decodestring'` specific error when python is 3.9

Reply:
wait so did you mean to have this be `3.7` instead of `2.7` ?

Reply:
yep it should be 3.7 now (i dont know if it refreshed in git)